specs/interop/storage-upgrade.md

```diff
@@ -76,6 +76,9 @@ function verifyAndStore(
 }
 ```
 
+The L2 `ProxyAdmin` owner will be responsible for setting the hash onion, on OP Mainnet this is the aliased L1
```

Comment:
The comma is weird. wdyt about something like this?

"Each L2 `ProxyAdmin` owner will be responsible for setting the hash onion. On OP Mainnet this address corresponds to the aliased L1 governance multisig."